### ISSUE TITLE
Undefined variable $node in _webform_import_import()

### ISSUE THREAD

**Issue by leeksoup** [backdrop-contrib/webform_import#5]
Get the following error messages on importing a CSV file of webform submissions:

```
Warning: Undefined variable $node in _webform_import_import() (line 378 of /var/www/backdrop1/modules/contrib/webform_import/webform_import.module).
Warning: Attempt to read property "webform" on null in webform_ip_address() (line 5249 of /var/www/backdrop1/modules/contrib/webform/webform.module).
Warning: Trying to access array offset on value of type null in webform_ip_address() (line 5249 of /var/www/backdrop1/modules/contrib/webform/webform.module).
```

$node really is not defined within that function nor passed into it. This leads to the other 2 errors, I believe. I'll try to fix this as well, since working on fixing https://github.com/backdrop-contrib/webform_import/issues/4

**Comment by herbdool** [comment]
@leeksoup thanks for reporting and the fix. Can you split your PR to cover each issue separately? It's hard to follow the PR as is.

If the maintainer isn't responding the @backdrop-contrib/bug-squad can merge bug fixes after they've been reviewed and tested. But it's more likely to get that if you split the PR. Thanks.

**Comment by leeksoup** [comment]
Yup, I can do that, I think. The second fix was only a one-line code change but I can separate it out if that helps.

**Comment by jenlampton** [comment]
@leeksoup yes please, that will allow the one-line changes to get in even if/when there is something wrong with the bigger changes. :)

**Comment by leeksoup** [comment]
Separate PR created as requested!

**Comment by herbdool** [comment]
The PR looks good to me.
